net/pfSense-pkg-cellular/files/etc/devd/cellular.conf

```diff
@@ -45,3 +51,18 @@ notify 1000 {
         action "%%PYTHON_CMD%% /usr/local/bin/cellular_dev.py -r";
 };
 
+# Quectel - EC25 LTE modem
+# ********
+attach 1000 {
+        match "vendor"          "0x2c7c";
+        match "product"         "0x0125";
+        action "/usr/local/bin/python3.8 /usr/local/bin/cellular_dev.py -a -d $device-name -m 2c7c0125";
```

Comment:
This should be using `%%PYTHON_CMD%%` and not the path to python directly.

Reply:
thanks for the comment. Updated